builds/ci/dotnet.yaml

```
@@ -74,6 +93,7 @@ jobs:
       vmImage: windows-2019
     variables:
       testEnvironment: windows
+      test.filter: Category=Integration&Category!=Stress&Category!=Kubernetes
```

why we do not use $(test.filter) variable here?
```suggestion
      test.filter: $(test.filter)&Category!=Kubernetes
```